teuthology/run_tasks.py

```diff
@@ -203,6 +204,41 @@ def run_tasks(tasks, ctx):
             del exc_info
         timer.mark("tasks complete")
 
+
+def build_rocket_message(ctx, stack, sleep_time_sec, template_path=None):
```

Comment:
I suggest as a less ambiguous name for the function: `build_rocket_chat_message`